Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -42,7 +42,8 @@
 namespace WebKit {
     
 static const Seconds processSuspensionTimeout { 20_s };
-static const Seconds removeAllAssertionsTimeout { 8_min };
+static const Seconds removeAllAssertionsTimeout { 1_min };
```

Comment:
This is confusing. This says we remove all assertions after 1 minute but the variable below says we're holding a temporary foreground assertion for 8 minutes.